openshift/tests-extension/pkg/helpers/pod_logs.go

```diff
@@ -0,0 +1,51 @@
+package helpers
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	//nolint:staticcheck // ST1001: dot-imports for readability
+	. "github.com/onsi/ginkgo/v2"
+)
+
+// GetAllPodLogs prints logs for all containers in all pods in the given namespace.
+func GetAllPodLogs(ctx context.Context, namespace string) {
+	fmt.Fprintf(GinkgoWriter, "\n[pod-logs] namespace=%s\n", namespace)
+
+	By("Getting all pods in the namespace")
+	namesOut, err := RunK8sCommand(ctx, "get", "pods", "-n", namespace, "-o", "name")
```

Comment:
Do we need a cli tool (kubectl/oc) to get pod logs? Can't we just use controller runtime client to get all the pods?

Reply:
QE tests will need to use the CLI so here we add the helper already for them
And in this case is more cleaner the code with oc/kubectl then with the API
I mean, the code is KISS. Otherwise, we need add more extra deps